lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -317,11 +317,11 @@ class SingleThreadQueryCountTestCase(ModuleStoreTestCase):
 
     @ddt.data(
         # old mongo with cache
-        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 12, 7),
-        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 12, 7),
+        (ModuleStoreEnum.Type.mongo, 1, 7, 5, 13, 7),
+        (ModuleStoreEnum.Type.mongo, 50, 7, 5, 13, 7),
```

Comment:
Thanks for confirming.  Although our performance cache epic will significantly optimize has_access, until then, we should stay at status quo with our mongo queries.  So please revert the order of operations in the 2 cases where staff is now checked first.